sci-physics/lhapdf/lhapdf-6.3.0-r2.ebuild

```
@@ -52,6 +52,7 @@ src_prepare() {
 }
 
 src_configure() {
+	CONFIG_SHELL="${EPREFIX}/bin/bash" \
```

```suggestion
	local -x CONFIG_SHELL="${EPREFIX}/bin/bash"
```

and then you can avoid a backslash.